client/jetpack-connect/style.scss

```diff
@@ -777,3 +776,167 @@ $colophon-height: 50px;
 .jetpack-connect__connect-button {
 	text-align: center;
 }
+
+/**
+ * Onboarding styles
+ **/
+.is-section-jetpack-connect:not(.is-add-site-page),
+.is-jetpack-login {
+	background-color: var( --color-jetpack-onboarding-background );
+
+	&.layout {
+		position: relative;
+		min-height: calc( 100% - #{$colophon-height} );
+		padding-bottom: $colophon-height;
+	}
+
+	.login__form-header,
+	.formatted-header {
+		color: var( --color-jetpack-onboarding-text );
+	}
+
+	.jetpack-logo__text,
+	.wpcom-colophon path {
+		fill: var( --color-jetpack-onboarding-text );
+	}
+
+	.formatted-header__subtitle,
+	.payment-methods,
+	.jetpack-connect__help-button {
+		color: var( --color-primary-100 );
+	}
+
+	.card,
+	.site-type__wrapper .card,
+	.site-topic__wrapper .card {
+		box-shadow: none;
+		border-radius: 3px;
+	}
+
+	.site__title:after,
+	.site__domain:after {
+		right: 3px;
+	}
+
+	.login__form-social {
+		margin-top: 1px;
+	}
+
+	.site-type__option-label {
+		color: var( --color-text );
+	}
+
+	.jetpack-connect__plan-info {
+
+		.formatted-header {
+			color: var( --color-text );
+		}
+
+		.formatted-header__subtitle {
+			color: var( --color-text-subtle );
+		}
+	}
+	
+	.wp-login__links a,
+	.logged-out-form__links a,
+	.logged-out-form__link-item {
```

Comment:
Seems like we've left out some links from this, specifically the "back" button in `/jetpack/connect/install`:

Regular:
![](https://cldup.com/Gpa6N0-lN3.png)

Hovered:
![](https://cldup.com/_cvB5MPJtO.png)

Reply:
Great catch! Done in https://github.com/Automattic/wp-calypso/pull/31141/commits/575c499cc61b1231cd52f1785921fe1ae1161a6e